src/icons/fileIcons.ts

```diff
@@ -2326,6 +2326,16 @@ export const fileIcons: FileIcons = {
       ],
     },
     { name: 'typst', fileExtensions: ['typ'] },
+    {
+      name: 'unocss',
+      fileNames: [
```

Comment:
The other variants with only `uno` instead of `unocss` should also be added:
```typescript
        'uno.config.ts',
        'uno.config.js',
        'uno.config.mts',
        'uno.config.mjs',
```

Also shouldn't they be alphabetically ordered (`js` before `ts`)?

Reference: <https://unocss.dev/guide/config-file>

PS: Thanks for your PR!

Reply:
Thanks for your comment. I will push new commit as soon as possible

Reply:
Maybe it would be good to put the `uno.` variants above the `unocss.` variants, so it's sorted from shortest to longest.
Otherwise this looks good.

Here is the code:
```typescript
      fileNames: [
        'uno.config.js',
        'uno.config.mjs',
        'uno.config.ts',
        'uno.config.mts',
        'unocss.config.js',
        'unocss.config.mjs',
        'unocss.config.ts',
        'unocss.config.mts',
      ],
```

Reply:
Thanks for your suggestion, I have pushed a new commit